pytition/pytition/settings/base.py

```diff
@@ -101,9 +101,25 @@
     }
 }
 
-if os.environ.get('USE_POSTGRESQL', ''):
+if os.environ.get('USE_POSTGRESQL'):
```

Comment:
Why is it necessary to remove the "empty string default value"?

Reply:
it is just that it is not useful, `get` will return `None` by default that will evaluate to `False`, so I though I'll remove it, but if you prefer I can add it back again

Reply:
Right, looks good then :)